src/eckit/log/SysLog.h

```diff
@@ -70,7 +71,7 @@ class SysLog {
     static char nilvalue() { return '-'; }
 
 
-    SysLog(const std::string& msg, int msgid = 0, Facility f = SysLog::User, Severity s = SysLog::Info);
+    explicit SysLog(std::string msg, int msgid = 0, Facility f = SysLog::User, Severity s = SysLog::Info);
```

Comment:
Given that we are going to be using this in a mode where the messages is set incrementally, it may make sense to have a constructor that does not require a message to create it initially.

Reply:
yes, it's annoyed me too. I tend to enlarge the PRs.. this time it's opposite